src/rebuild/srv.c

```diff
@@ -1134,6 +1134,12 @@ rebuild_task_ult(void *arg)
 		return;
 	}
 
+	rc = rebuild_notify_ras_start(&task->dst_pool_uuid,
+				      RB_OP_STR(task->dst_rebuild_op));
```

Comment:
please add pool map version here as well. thanks.

Reply:
Do you think the map version needs to be part of the event structure or is it OK to put it in the message? I guess the question is: Do you foresee the version being a key when searching events, or is it just to make the message more informative?

Reply:
yes, we use pool uuid + version to identify one rebuild.  With the version, we can know which are failure targets for the rebuild.

Reply:
Thanks. I've added the version as part of the structured data.